cscs-checks/mch/alltoallv.py

```diff
@@ -0,0 +1,95 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.parameterized_test([''], ['--nocomm'], ['--nocomp'])
+class Alltoallv(rfm.RegressionTest):
+    def __init__(self, exec_parameter):
+        super().__init__()
+        self.valid_systems = ['daint:gpu', 'dom:gpu', 'kesch:cn']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        if self.current_system.name in ['daint', 'dom']:
+            self.modules = ['craype-accel-nvidia60']
+            self._pgi_flags = '-acc -ta=tesla:cc60 -Mnorpath'
+        elif self.current_system.name in ['kesch']:
+            self.modules = ['craype-accel-nvidia35']
+            self._pgi_flags = '-O2 -ta=tesla,cc35,cuda8.0'
+
+        self.num_tasks = 144
+        self.num_gpus_per_node = 16
+        self.num_tasks_per_node = 16
+        self.num_tasks_per_socket = 8
+        self.executable = 'src/comm_overlap_benchmark %s' % exec_parameter
```

Comment:
Use `self.executable_opts` to pass the parameters.

Reply:
done